includes/machine-learning-data-labeling-describe.md

```diff
@@ -9,9 +9,12 @@ ms.author: sdgilley
 It's important to clearly explain the labeling task. On the **Labeling instructions** page, you can add a link to an external site for labeling instructions, or provide instructions in the edit box on the page. Keep the instructions task-oriented and appropriate to the audience. Consider these questions:
 
 * What are the labels they'll see, and how will they choose among them? Is there a reference text to refer to?
+* What is the visual definition of each label class? Is it possible to provide a list of normal, edge and counter cases for each class? 
```

Comment:
This set of questions is used in both the image labeling and text labeling document.  So talking about the "visual" definition is not appropriate here.  Can you move this to the image-labeling file instead?  OR take out the word "visual" and leave it here.

Reply:
I have shifted this to image-labeling docs. please review